### PR TITLE
Fix 6hr timeout on e2e integration tests

### PR DESCRIPTION
**Pull request by huutomerkki**
As title says.

### CODE REVIEW

**Review by mboukhalfa — APPROVED** [review]
/lgtm

**Comment by mboukhalfa** [comment]
/assign @kashifest
cc @furkatgofurov7

File: jenkins/jobs/integration_tests.pipeline [hunk 1]
```diff
@@ -9,7 +9,7 @@ def TIMEOUT = 7200
 KEEP_TEST_ENV=(env.KEEP_TEST_ENV)
 
 if (env.TESTS_FOR) {
-  if ( (env.TESTS_FOR).startsWith('feature_tests') || (env.TESTS_FOR).startsWith('e2e_tests') ) {
+  if ( (env.TESTS_FOR).startsWith('feature_tests') || ( (env.TESTS_FOR).startsWith('e2e_tests') && !(env.GINKGO_FOCUS).equals('integration'))) {
```
Comment (mboukhalfa) [inline jenkins/jobs/integration_tests.pipeline:12]: /hold
@huutomerkki We might need basic here as well

Comment (huutomerkki) [reply jenkins/jobs/integration_tests.pipeline]: Added it as well

**Pull request merged**